django_geomultiplechoice/widgets.py

```diff
@@ -30,13 +30,31 @@ def get_context(self, name, value, attrs):
         """
         context = super().get_context(name, value, attrs)
 
-        if self.settings_overrides.get('MAP_ID'):
-            context['MAP_ID'] = self.settings_overrides['MAP_ID']
-        else:
-            context['MAP_ID'] = 'map'
+        MAP_LAYER_STYLE = {
+          'color': '#7a7a7a',
+          'weight': 3,
+          'opacity': 0.5,
+          'fillColor': '#999999',
+          'fillOpacity': 0.3,
+        }
+
+        MAP_LAYER_SELECTED_STYLE = {
+          'color': '#7a7a7a',
+          'weight': 3,
+          'opacity': 0.5,
+          'fillColor': 'black',
+          'fillOpacity': 0.7
+        }
+
+        settings_defaults = [
+            ('MAP_ID', 'map'),
+            ('MAP_LAYER_STYLE', MAP_LAYER_STYLE),
+            ('MAP_LAYER_SELECTED_STYLE', MAP_LAYER_SELECTED_STYLE)
+        ]
 
-        context['MAP_WIDTH'] = self.settings_overrides.get('MAP_WIDTH')
-        context['MAP_HEIGHT'] = self.settings_overrides.get('MAP_HEIGHT')
+        for variable, default in settings_defaults:
```

Comment:
@jeancochrane I'm curious to hear what syntax you'd use here, as I can think of several ways of setting these default values

Reply:
I like your solution! In the future we may want to add validation to ensure that the format of the variables matches what we expect, which would require refactoring this assignment loop, but for now I think it's a nice and compact solution.